src/utils/format/DateHelper.ts

```diff
@@ -1,6 +1,15 @@
 export const timestampToString = (timestamp: number) => {
+
+  if (!timestamp || isNaN(timestamp)) {
+    return "Date invalide";
+  }
+
   const date = new Date(timestamp);
   const today = new Date();
+  
+  if (isNaN(date.getTime())) {
```

Comment:
Perso, je préfère comme ceci (je l'utilise comme ceci sur mon projet perso TBpM)

```suggestion
  if (Number.isNaN(date.getTime())) {
```